suites/nautilus/upgrades/multihop_upgrade_to_4.x.yaml

```diff
@@ -0,0 +1,81 @@
+# Upgrade: Multihop upgrade evaluation
```

Comment:
Could we rename the suite file to `<stage-name>_<upgrade>_test-multi-path-upgrade-to-4-latest.yaml`? This would enable us to adhere to the guidelines.

Reply:
Done.